calcom/cal.com
# Attach credential id to destinationCalendar records

## What does this PR do?



This PR address a bug where if a user had multiple Google Cal accounts connect to their account bookings would be created on all accounts.

Before when creating calendar events it would grab all the user's calendar credentials and only create calendar events on credentials that would match the destination calendar. This is why the bug only appeared with two Google Cals and not one Google Cal and an Outlook credential.

This PR adds the credential ID to the destination calendar record so when creating bookings, rather than looping through all credentials from the user. This should also increase booking performance.

Fixes # (issue)



**Environment**: Staging(main branch) / Production

## Type of change




## How should this be tested?



- Connect two Google Calendar accounts
- Create a booking
- When looking at the calendars of both accounts, the event should have only been created on the destination calendar

## Checklist


- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes
